src/Wordpress/CustomPostType/Map.php

```diff
@@ -20,355 +18,610 @@ class Map extends CustomPostType {
+				'desc'       => esc_html__( 'the text for the button used for filtering', 'commonsbooking' ),
+				'id'         => 'custom_filterbutton_label',
+				'type'       => 'text',
+				'attributes' => array(
+					'placeholder' => esc_html__( 'filter', 'commonsbooking' ),
+				)
+			),
+			array(
+				'name'       => commonsbooking_sanitizeHTML( __( 'Filter groups', 'commonsbooking' ) ),
+				'desc'       => commonsbooking_sanitizeHTML( __( 'Filter groups can group item or location categories together to allow for filtering in the map.', 'commonsbooking' ) ),
+				'id'         => 'filtergroups',
+				'type'       => 'group',
+				'repeatable' => true,
+				'options'    => array(
+					'group_title'   => commonsbooking_sanitizeHTML( __( 'Filter group {#}', 'commonsbooking' ) ),
+					'add_button'    => commonsbooking_sanitizeHTML( __( 'Add another filter group', 'commonsbooking' ) ),
+					'remove_button' => commonsbooking_sanitizeHTML( __( 'Remove filter group', 'commonsbooking' ) ),
+					'sortable'      => true,
+				),
+				'fields'     => array(
+					array(
+						'name'    => commonsbooking_sanitizeHTML( __( 'Name', 'commonsbooking' ) ),
+						'id'      => 'name',
+						'type'    => 'text',
+						'desc'    => commonsbooking_sanitizeHTML( __( 'The name of the filter group', 'commonsbooking' ) ),
+						'default' => '',
+					),
+					array(
+						'name'       => commonsbooking_sanitizeHTML( __( 'Type', 'commonsbooking' ) ),
+						'id'         => 'type',
+						'type'       => 'select',
+						'desc'       => commonsbooking_sanitizeHTML( __( 'This is not available yet. Which data source should be used for the filter group. Taxonomy stands for the assigned categories, post-meta can be individually configured custom fields for item posts. When this item field contains data, it will be included. If it does not contain data or does not exist, the item will be excluded.', 'commonsbooking' ) ),
+						'options'    => array(
+							'taxonomy' => commonsbooking_sanitizeHTML( __( 'Taxonomy', 'commonsbooking' ) ),
+							'postmeta' => commonsbooking_sanitizeHTML( __( 'Post-meta', 'commonsbooking' ) ),
+						),
+						'default'    => 'taxonomy',
+						//TODO: disabled until postmeta is implemented
+						'attributes' => array(
+							'disabled' => true
+						),
+					),
+					array(
+						'name' => commonsbooking_sanitizeHTML( __( 'Exclusive selection', 'commonsbooking' ) ),
+						'id'   => 'isExclusive',
+						'type' => 'checkbox',
+						'desc' => commonsbooking_sanitizeHTML( __( 'WARNING: This feature is only available for the cb_search shortcode, not for cb_map. If checked, only one category can be selected in this filter group. If unchecked, multiple categories can be selected.', 'commonsbooking' ) ),
+					),
+					array(
+						'name'              => commonsbooking_sanitizeHTML( __( 'Categories', 'commonsbooking' ) ),
+						'id'                => 'categories',
+						'type'              => 'multicheck',
```

Comment:
wieso hast du hier eigentlich multicheck und nicht `taxonomy_multicheck` verwendet?

https://cmb2.io/docs/field-types#-types-taxonomy_multicheck

Diesen Typ gibt es in CMB2 auch. Vielleicht weil es erst nochmal durch `sanitizeOptions` geht?

Reply:
Ich glaube das war der Grund, ja! An anderen Stellen wird das auch genutzt und ich bilde mir ein das auch ohne probiert zu haben und an irgendwas hing es. Vielleicht an der Art und Weise wie es dann gespeichert wurde.

Reply:
Ah ja, der Grund dafür ist auch, dass wir da in Zukunft noch Standortkategorien hinzufügen können.